Dockerfile

```diff
@@ -15,8 +15,14 @@ COPY charts /charts
 COPY --from=builder /go/bin/gardener-extension-provider-gcp /gardener-extension-provider-gcp
 ENTRYPOINT ["/gardener-extension-provider-gcp"]
 
-############# gardener-extension-validator-aws
+############# gardener-extension-admission-gcp
+FROM base AS gardener-extension-admission-gcp
+
+COPY --from=builder /go/bin/gardener-extension-admission-gcp /gardener-extension-admission-gcp
+ENTRYPOINT ["/gardener-extension-admission-gcp"]
+
```

Comment:
Can you remove everything below this line?

Reply:
I hadn't specified the "old" validator image but then struggled with a failing CI pipeline since the new definition is not effective as long as the PR is not merged. Hence, ddd8e23cd6ddad2403be89f32d5760387aa9e873 adds the old image again and can be reverted once the changes are merged to the master branch. Do you have another idea how to handle this situation, so I can remove the mentioned lines directly?

Reply:
OK, thanks, seems to be the most pragmatic way. Then let's merge this PR, can you open a new one that reverts the commit again?

Reply:
https://github.com/gardener/gardener-extension-provider-gcp/pull/92